.travis.yml

```diff
@@ -1,18 +1,21 @@
 language: node_js
 node_js:
-  - "8"
-  - "9"
-  - "10"
-  - "12"
+  - 8
+  - 9
```

Comment:
Can we remove 9 to solve the build issues? It is not necessary anyways.

Reply:
clearing the travis cache resolves this